extensions/amp-ad/0.1/amp-ad.css

```diff
@@ -48,9 +48,13 @@ amp-embed iframe {
   padding: 2px 4px;
 }
 
-amp-ad[type=adsense] > iframe {
+amp-ad[data-a4a-upgrade-type] > iframe {
+  width: 100%;
+  height: 100%;
   top: 50% !important;
   left: 50% !important;
   transform: translate(-50%, -50%);
+  min-width: 0;
+  min-height: 0;
```

Comment:
Manual tests on Chrome appear fine. I will coordinate with someone who has a Mac to test on Safari.